README.md

```diff
@@ -24,19 +24,46 @@ pyuvsim's two primary goals are interferometer simulation accuracy at the level
 6. Support for varied beam models across the array.
 7. Defining a clear, user-friendly standard for simulation design.
 
-# Installation
-* For simple installation, the latest stable release is available via pip (`pip install pyuvsim`)
-* To install the development version: Clone the repository using `git clone https://github.com/RadioAstronomySoftwareGroup/pyuvsim`, navigate into the pyuvsim directory and run
-```
-pip install -r requirements.txt
-pip install pyuvdata
-pip install .
-```
-* `pyuvsim` is intended to run on clusters running the linux operating system.
+## Installation
+A user-installation is achieved simply with `pip install pyuvsim`, or to get the 
+bleeding-edge: `pip install https://github.com/RadioAstronomySoftwareGroup/pyuvsim`. 
+This will install all dependencies. 
+
+If you wish to manage dependencies manually, or are developing `pyuvsum` yourself, read
```

Comment:
`pyuvsum` -> `pyuvsim`

Reply:
Thanks. Fixed in 234179010bd3ce72ff46e09aab3327e4b2dac78d